project_recalculate/models/resource_calendar.py

```diff
@@ -0,0 +1,74 @@
+# See README.rst file on addon root folder for license details
+
+from odoo import api, models
+from datetime import datetime, timedelta
+
+from odoo.addons.resource.models.resource import make_aware
+from functools import partial
+
+
+class ResourceCalendar(models.Model):
+    _inherit = 'resource.calendar'
+
+    def get_working_days_of_date(self, start_dt=None, end_dt=None,
+                                 resource=None):
+        self.ensure_one()
+        if start_dt is None:
+            start_dt = datetime.now().replace(hour=0, minute=0, second=0)
+        if end_dt is None:
+            end_dt = datetime.now().replace(hour=23, minute=59, second=59)
+        days = 0
+        current = start_dt
+        while current <= end_dt:
+            if id is None:
```

Comment:
:warning: [`id` is a built-in function](https://docs.python.org/3/library/functions.html#id) that will never be `None`. I'm not sure what this code was supposed to do, but it's wrong. :sweat_smile: